### PR TITLE
Merge dotnet_ui_2.0-to-master (#1445) to tizen_6.5_prepare

### PR DESCRIPTION
**Pull request by piotrekgorny**
This PR merges the dotnet_ui_2.0-to-master branch (please refer to #1445 PR) to the tizen_6.5_prepare branch (with conflicts resolving).

### CODE REVIEW

**Comment by annie-samsung** [comment]
@mijin85cho, @piotrekgorny since this PR is reviewed multiple times, can I just approve and merge it? Do I need to perform any specific checks?

**Comment by piotrekgorny** [comment]
@annie-samsung 
I think that someone of "tizen_6.5_prepare" branch's creators should check if we properly resolved conflicts with their changes.

**Review by mijin85cho — COMMENTED** [review]
@annie-samsung  master branch and tizen_6.5_prepare branch have differences in history, thus some files are not same.
Also it seems @piotrekgorny also merged some other updates in master to tizen_6.5_prepare,
as you can see that in https://github.com/Samsung/tizen-docs/pull/1445, there are 316 updated files,
but in https://github.com/Samsung/tizen-docs/pull/1483, 391 files.
(Thanks for that, @piotrekgorny )

It seems at least checking the difference between #1445  and #1483 would be needed.

File: docs/application/dotnet/guides/nui/overview.md
Comment (annie-samsung) [reply docs/application/dotnet/guides/nui/overview.md:46]: @mijin85cho, yes in PR 1480 I had raised the comment and found that overview file was different, however that seems to be fixed now in 1483, so post this is merged I was thinking to update myself or ask the 1480 developer to update overview file again.

**Comment by annie-samsung** [comment]
> @annie-samsung master branch and tizen_6.5_prepare branch have differences in history, thus some files are not same.
> Also it seems @piotrekgorny also merged some other updates in master to tizen_6.5_prepare,
> as you can see that in #1445, there are 316 updated files,
> but in #1483, 391 files.
> (Thanks for that, @piotrekgorny )
> 
> It seems at least checking the difference between #1445 and #1483 would be needed.

@mijin85cho, okay, I will start checking the diff files

**Comment by annie-samsung** [comment]
> @annie-samsung
> I think that someone of "tizen_6.5_prepare" branch's creators should check if we properly resolved conflicts with their changes.

@piotrekgorny 
I will try and check as much as possible, however this is  going to be challenging as we delay merging these files more conflicts would arise.

File: docs/application/dotnet/guides/applications/uiapplication/component-based-app.md
Comment (piotrekgorny) [reply docs/application/dotnet/guides/applications/uiapplication/component-based-app.md:307]: It looks like this guide was copied from the native part but links weren't updated. These are links to the native API and should be updated to refer TizenFX API.

Comment (piotrekgorny) [reply docs/application/dotnet/guides/applications/uiapplication/component-based-app.md:307]: I mean that before this PR they were incorrect so we left them as they were.

**Comment by journey2w** [comment]
To create a staging URL, https://docs1.stg.tizen.org/staging/1483, I'm going to close and then reopen it.

**Review by sagarkedari — CHANGES_REQUESTED** [review]

File: docs/application/toc_all.md
Comment (mijin85cho) [reply docs/application/toc_all.md:130]: Hello, @annie-samsung 

Would you please guide @sagarkedari, about the md file structure and the URL relation?

@sagarkedari, md files structure becomes URL of each page. 
As you can see that the links in left navigation bar are working well, there is no need to use same folder name & menu name.

If we use same folder name to the LNB menu, it would be hard to understand what that section is about.
Likewise that, if you use exact menu name to the folder, the folder path would be too long, which will results to too long URL.

Comment (annie-samsung) [reply docs/application/toc_all.md:130]: @mijin85cho sure will keep that in mind. We have identified the MD files and did not find any content change from 1387 to 1483. For 1387 we used the list Nawab prepared. I will be doing some random grep checks by Monday or Tuesday we shall be posting our final feedback. We are now checking images.

**Comment by annie-samsung** [comment]
@piotrekgorny. @mijin85cho 

I have completed my review here are my observations and approach:

1) I compared the MD files of 1483 and 1387 (excel that Nawab had then prepared), I could not find any additional MD files in 1483. The only major change I could find was with styling_with_xaml.md  and that is because there has been changes already done to that file in 6.5 branch.

If I have missed to identify MD files that you specifically modified for 1483 (not part of 1387) request you to kindly help with the list.

2) I have checked for images cross reference via Grep especially for folders changes (like Personal, Certificates, account, libraries, sensors). I did not find any issues in the images, except the following images that are not referenced in any file:

pims/media/calendar.png
pims/media/calendar_property.png
pims/media/calendar_style.png
pims/media/calendar_children.png
pims/media/contacts_unlink_contact.png

3) For NUI, I see left Nav  and URL following a different structure, while I understand it is not mandatory to follow the same structure. The rest of the sections all follow the same structure as left Nav for URL as well (folder structure).

I think we can ignore it as changing then can again lead to broken links etc.


If I missed out any checks please let me know.

**Comment by mijin85cho** [comment]
@annie-samsung Thanks for your checking. I appreciate your effort.

For the second issue, unused image files, my opinion is, 
if there are any files which are not in use, it is strongly recommended to remove them. 
However this PR is for applying .NET Guide structure update, so how about creating another PR to remove those images to make this PR focus on the purpose?

**Comment by annie-samsung** [comment]
> @annie-samsung Thanks for your checking. I appreciate your effort.
> 
> For the second issue, unused image files, my opinion is,
> if there are any files which are not in use, it is strongly recommended to remove them.
> However this PR is for applying .NET Guide structure update, so how about creating another PR to remove those images to make this PR focus on the purpose?

@mijin85cho sure I shall raise a new PR for images that are not used. 

@piotrekgorny do you have any comments? Also, I now see folder structure change introduced for NUI guides in 1491 and 1492 could you please check on the same and let me know your thoughts?

**Comment by annie-samsung** [comment]
@piotrekgorny, gentle reminder!

**Comment by piotrekgorny** [comment]
@annie-samsung 

I'm sorry but I'm out of office till August 24th and I don't have access to computer to check it out.

**Comment by annie-samsung** [comment]
> @annie-samsung
> 
> I'm sorry but I'm out of office till August 24th and I don't have access to computer to check it out.

Okay  I shall wait for your response to further approve or merge this PR.

**Comment by piotrekgorny** [comment]
> If I have missed to identify MD files that you specifically modified for 1483 (not part of 1387) request you to kindly help with the list.

Below is the list of files specifically modified in 1483 (copied from gitk):

docs/application/dotnet/guides/applications/uiapplication/component-based-app.md
docs/application/dotnet/guides/applications/uiapplication/media/application_lifecycle.jpg
docs/application/dotnet/guides/applications/uiapplication/media/frame_lifecycle.jpg
docs/application/dotnet/guides/applications/uiapplication/media/service_lifecycle.jpg
docs/application/dotnet/guides/applications/uiapplication/overview.md
docs/application/dotnet/guides/nui/customview.md
docs/application/toc_all.md

**Comment by piotrekgorny** [comment]
> Also, I now see folder structure change introduced for NUI guides in 1491 and 1492 could you please check on the same and let me know your thoughts?

I think that these changes are probably reasonable but they should  be applied after merging this PR (I do not know if I will be able to resolve conflicts if they will be merged first)

**Comment by annie-samsung** [comment]
> overview.md

Hi, I have checked the changes, some are just file renaming and cutom view is link change which looks fine.
I am going ahead and approving and merging this PR

**Review by annie-samsung — APPROVED** [review]
Approving

**Comment by annie-samsung** [comment]
@piotrekgorny, there is some conflict in Visuals.md could you please resolve it, so that I can merge this PR

**Comment by piotrekgorny** [comment]
> @piotrekgorny, there is some conflict in Visuals.md could you please resolve it, so that I can merge this PR

Done

**Pull request merged**
